termbox/compat.go

```diff
@@ -346,6 +365,11 @@ func PollEvent() Event {
 	return makeEvent(ev)
 }
 
+func PollEventOnScreen(s tcell.Screen) Event {
```

Comment:
This would be another thing to move to another namespace to avoid polluting the termbox namespace.